data/human/hails.txt

```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
 		"neodymium and neodymium" "neodymium"
 		"neodymium or neodymium" "neodymium"
 
+phrase "friendly civilian"
+	word
+		"Be careful of"
+		"A warning about"
+		"Be suspicious of"
+		"Don't trust"
+	word
+		" "
+	word
+		"those space casinos. I blew"
+	word
+		" "
+	word
+		"half a"
+		"a"
+		"two"
+		"three"
+		"four"
+		"five"
+	word
+		" million credits on "
+	word
```

Comment:
Is there any particular reason that we have:
```
	word
		" "
	word
		"those space casinos. I blew"
	word
		" "
	word
```
And then also:
```
	word
		" million credits on "
	word
```
?
They appear to be doing the same thing (this part of the result from this phrase should always be this series of words, no other options) but the first puts the leading trailing spaces in their own "word"s while hte second includes them in the "word" with the other words.